recipe/meta.yaml

```diff
@@ -21,38 +21,52 @@ requirements:
     - python                                 # [build_platform != target_platform]
     - cross-python_{{ target_platform }}     # [build_platform != target_platform]
     - numpy                                  # [build_platform != target_platform]
+    - pyside6
     - {{ compiler('cxx') }}
   host:
     - python
     - pip
-    - numpy <1.24
-    # Maintainer note: remove this numpy pin when sasview 5.0.6 is released, 5.0.6 and newer patch this issue.
-    - pyqt >=5.12
+    - numpy ==1.26
+    - pyside6 ==6.4
+    - scipy ==1.13
```

Comment:
@wpotrzebowski, are there any particular reasons to hard-pin those dependencies? Maybe we can relax them to have lower/upped bounds (e.g., `numpy <2`, `scipy >=1.13`, etc.)?

Reply:
we only need 3 pinnings, one of them being hard, due to PySide >6.4 issues with pyinstaller.
This can be changed to:
```python
numpy <2
PySide ==6.4
scipy < 1.14
```